scripts/rendering-tests-reference-platform.sh

```diff
@@ -3,16 +3,16 @@
 #
 # Run harp.gl rendering tests (see [../test/README.md]) on reference platform i.e
 #
-#   Headless Chrome 78 on Linux with GPU disabled
+#   Headless Chrome 80 on Linux with GPU disabled
 #
 
 #
 # Docker image with chrome
 #
 # See https://github.com/SeleniumHQ/docker-selenium/releases.
 #
-# xenon stands for chrome 78
-REFERENCE_IMAGE=selenium/standalone-chrome:3.141.59-xenon
+# zirconium stands for chrome 80
+REFERENCE_IMAGE=selenium/standalone-chrome:3.141.59-zirconium
```

Comment:
`REFERENCE_IMAGE` is a quite bad name for the Docker image b/c especially for IBCT reference image is already occupied.